ROMFS/px4fmu_common/init.d/4900_crazyflie

```diff
@@ -45,21 +54,24 @@ then
 	param set MPC_Z_VEL_P 0.4
 	param set MPC_HOLD_MAX_XY 0.1
 	param set MPC_MAX_FLOW_HGT 3
-	param set IMU_GYRO_CUTOFF 100
-	param set IMU_ACCEL_CUTOFF 30
-	param set MC_DTERM_CUTOFF 70
-	param set SYS_FMU_TASK 1
-	param set CBRK_SUPPLY_CHK 894281
-	param set CBRK_USB_CHK 197848
-	param set SDLOG_PROFILE 1
-	param set EKF2_MAG_TYPE 1
-	param set EKF2_ABL_LIM 2.0
-	param set MC_AIRMODE 1
+
 	param set NAV_RCL_ACT 3
+
+	param set PWM_DISARMED 0
+	param set PWM_MIN 0
+	param set PWM_MAX 255
+
+	# Run the motors at 328.125 kHz (recommended)
+	param set PWM_RATE 3921
+
+	param set SDLOG_PROFILE 1
 	param set SENS_FLOW_MINRNG 0.05
+	param set SYS_COMPANION 20	
+	param set SYS_FMU_TASK 1
+
 fi
-set PWM_MIN none
-set PWM_MAX none
+
+# TODO - These values overwrite param sets above.  Can they be removed?
```

Comment:
Crazyflie is a bit special and I'm not sure if we can remove this. I suggest leaving it in for now.